src/types.rs

```diff
@@ -15,6 +17,12 @@ pub struct Coordinate<T>
     pub y: T,
 }
 
+impl<T: Float> From<(T, T)> for Coordinate<T> {
```

Comment:
I'm _very_ hesitant about adding this implementation. Projects are not consistent about `(x, y)` vs. `(y, x)`: https://macwright.org/lonlat/